content/docs/2.11/operate/prometheus.md

```diff
@@ -16,7 +16,7 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 - `keda_scaler_metrics_latency` - The latency of retrieving current metric from each scaler.
 - `keda_scaler_errors` - The number of errors that have occurred for each scaler.
 - `keda_scaler_errors_total` - The total number of errors encountered for all scalers.
-- `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObject.
+- `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObejct.
```

Comment:
why?

Reply:
Didn't add the line. i think git decided to treat it as a new change.

Reply:
```suggestion
- `keda_scaled_object_errors` - The number of errors that have occurred for each ScaledObject.
```